vm-setup/roles/libvirt/defaults/main.yml

```diff
@@ -16,6 +16,42 @@ libvirt_arch: x86_64
 libvirt_cpu_mode: host-model
 libvirt_firmware: bios
 
+#EPHEMERAL_CLUSTER variable
+EPHEMERAL_CLUSTER: "{{ lookup('env', 'EPHEMERAL_CLUSTER')}}"
+
+#baremetal_interface variable
+baremetal_interface: baremetal
+
+provisioning_interface: provisioning
+
+vbmc_port_range: "6230:6235"
+ironic_ports:
+  - "6180"
+  - "5050"
+  - "6385"
+  - "9999"
+provisioning_host_ports:
+  # Caching HTTP Server
+  - "80"
+  # Container image registry
+  - "5000"
+  # DNS for registry naming resolution
+  - "53"
+pxe_udp_ports:
+  # Multicast DNS
+  - "5353"
+  # DHCP
+  - "67"
+  - "68"
+  # DHCPv6
+  - "546"
+  - "547"
+  # TFTP
+  - "69"
+ironic_keepalived_proto:
+  - "112"
+  - "icmp"
+
```

Comment:
I am not so in-favor of keeping the same info in two places. That would require back-porting each and every change and its difficult to manage. Is it possible to define this in one place and then let firewall and libvirt tasks source it?

Reply:
I do agree.  I can take a look on this issue..